example/src/screens/CustomerSheetScreen.tsx

```diff
@@ -68,7 +68,7 @@ export default function CustomerSheetScreen() {
       setupIntentClientSecret,
       customerEphemeralKeySecret,
       customerId,
-      returnURL: 'stripe-example://stripe-redirect',
+      returnURL: 'com.stripe.react.native://stripe-redirect',
```

Comment:
I missed some when updating the example app, the new linking scheme should be com.stripe.react.native and not stripe-example.